### DISCUSSION TITLE
Copy to Clipboard buttons are not working

### DISCUSSION THREAD

**Discussion by CBB3810** [pglombardo/PasswordPusher#1251] [Q&A] [answered]
Is there something extra I need to do in order to enable the "Copy to Clipboard" buttons?

**Comment by CBB3810** [comment, marked as answer]
Closing, it's working now. I don't even know why but I'm not going to tempt fate. Thanks for the use of your software.

**Reply by pglombardo** [reply]
Hi @CBB3810 - glad to hear it's working now.  I do know that there is some copy to clipboard browser limitation on non SSL host sites (if that is your case).  See https://github.com/pglombardo/PasswordPusher/issues/734 for gory details if interested.
